### DISCUSSION TITLE
Proposed documentation update

### DISCUSSION THREAD

**Discussion by jacobstim** [Thomas55555/husqvarna_automower#354] [General]
Hi all,

Ok, feeling slightly dumb but I suppose this is a mistake other people might make too :). In the readme.md, it says

> The camera entity is configured via the configure option on the integration. To enter the coordinates, ensure that they are in Signed Degree format and separated by a comma for example (40.689209, -74.044661)

I propose to add to this that the **brackets** should not be included in the fields, i.e. that it is just two floats separated by a comma. Adding brackets would throw "an unknown error" in the integration and log in Home Assistant that "(XX.XXXX cannot be converted from string to float". I guess most users would be able to troubleshoot this but well, a small addition in the docs might save them those 3 minutes ;).

**Comment by mulcmu** [comment]
PR #355 places the example coordinates in a code block for clarity.  

![Untitled](https://user-images.githubusercontent.com/10102873/193306804-85abe08b-a8d6-4ed9-9fd4-f6b85d13112f.png)
